examples/heartbeats.py

```diff
@@ -0,0 +1,45 @@
+import unicornhathd as unicorn
+import time
+import colorsys
+import numpy
+
+unicorn.brightness(1)
+# need to rotate the image to have the heart the right way up
+unicorn.rotation(90)
+
+heart = [[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0],
+         [0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0],
+         [0, 1, 1, 1, 1, 1, 1, 0, 0, 1, 1, 1, 1, 1, 1, 0],
+         [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+         [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+         [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+         [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+         [0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0],
+         [0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0],
+         [0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0],
+         [0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]]
+
+heart = numpy.array(heart)
+
+while True:
+    #  go through the range 1-10 backwards, then back up
+    #  the 2* makes a ba-BUMP for the heart
+    for i in 2*(range(1, 11)[::-1]+range(1, 10)):
+        for y in range(16):
+            for x in range(16):
+                h = 0.0  # red
+                s = 1.0  # saturation at the top of the red scale
+                v = heart[x, y] / float(i)  # brightness depends on range
+                rgb = colorsys.hsv_to_rgb(h, s, v)  # convert hsv back to RGB
```

Comment:
Since `hsv_to_rgb` returns a tuple, you can assign it to 3 different variables right from the get go:

```suggestion
                r, g, b = colorsys.hsv_to_rgb(h, s, v)  # convert hsv back to RGB
```

Reply:
That is very handy. Noted and updated.